### DISCUSSION TITLE
Mixer - Multiply question

### DISCUSSION THREAD

**Discussion by Aclarky1969** [pschatzmann/arduino-audio-tools#199] [Q&A] [answered]
Hi
Firstly, thank you for a wonderful set of Arduino audio tools.

I have used the Teensy audio design tool to make a SDR transceiver and was wondering if it's possible to
have a mixer/summer and multiplier function. I'm planning on doing the same with a ESP-32.

Regards

Alan

**Comment by pschatzmann** [comment]
I am not sure what exactly this should do. Maybe you need to be a bit more specific:
I currently have 
- https://pschatzmann.github.io/arduino-audio-tools/html/classaudio__tools_1_1_mixer.html
- https://pschatzmann.github.io/arduino-audio-tools/html/classaudio__tools_1_1_converter_scaler.html
- 
And you can use the full functionality of other frameworks like [Maximilian](https://github.com/pschatzmann/Maximilian), [Faust](https://faust.grame.fr/) etc

**Reply by veitalmaphil** [reply]
do you use the mixer function in any example? 
i'm trying to play two sounds independently for a game ...

i just found your mixer-blog-entry and will read before i ask :)

**Comment by pschatzmann** [comment, marked as answer]
Right, here is the [link to the example](https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-audiokit/streams-generator_mixer-audiokit/streams-generator_mixer-audiokit.ino)

See also the related documentation: https://github.com/pschatzmann/arduino-audio-tools/wiki/Splitting-and-Merging-Audio
